keras_cv/__init__.py

```diff
@@ -23,6 +23,7 @@
 from keras_cv import losses
 from keras_cv import metrics
 from keras_cv import models
+from keras_cv import training
```

Comment:
We should probably discuss more on having a training directory

Reply:
Agreed. I'm not certain what the right namespacing is, but I thought this was a good starting point for the conversation.

Reply:
@fchollet any opinion here?

Reply:
from design review:  training is a good namespace.

Lets resolve!